packages/extension/src/standalone/standalone.js

```diff
@@ -5,6 +5,34 @@ import App from './App.vue';
 import store from '../store';
 import { getCache, STATE_KEY } from '../common/cache';
 import { debug } from '../common/config';
+import { browserName } from '../common/browser';
+
+const loadFromCache = async () => {
+  const state = await getCache(STATE_KEY, {});
+
+  if (!store.state.initialized) {
+    store.commit('loadFromCache', state);
+  }
+
+  const source = window.location.href.split('source=')[1];
+  if (!source && state.ui && new Date().getTime() <= state.ui.dndModeTime) {
+    const tab = await browser.tabs.getCurrent();
+    const url = browserName === 'chrome'
+      ? 'chrome-search://local-ntp/local-ntp.html'
+      : 'https://www.google.com';
+
+    browser.tabs.update(tab.id, { url });
+    window.stop();
+  } else {
+    document.documentElement.classList.add('loaded');
+
+    if (state.ui.dndModeTime) {
+      store.commit('ui/disableDndMode');
```

Comment:
This will be called not only when dnd is over but also if I clicked the Daily button, am I right? I think it's a bug